### DISCUSSION TITLE
Include docker-slim in Drone CI manifest

### DISCUSSION THREAD

**Discussion by btalberg** [slimtoolkit/slim#241] [Show and tell]
We use [Drone](https://www.drone.io/) as our CI platform. I added a docker-slim step to shrink our image and push it up to the Hub registry. If others are interested, take a look here: https://github.com/WildflowerSchools/wf-drone-slim-template/blob/main/drone.yml#L33. PS: You can see the Docker Hub tags that are created with this YML recipe at our [respective hub repo](https://hub.docker.com/r/wildflowerschools/wf-drone-slim-template/tags).

Open to feedback and improvement ideas!

**Comment by kcq** [comment]
This is awesome! Thank you for sharing @btalberg !
